playwright-e2e/dss/pages/atcp/atcp-dashboard-page.ts

```diff
@@ -13,7 +13,6 @@ export default class AtcpDashboardPage extends AtcpPageBase {
     await super.waitForReady();
     await expect(this.page.locator('h1.title')).toHaveText('Thank you for joining the Global A-T Family Data Platform!');
     await waitForNoSpinner(this.page);
-    await this.getTable().waitForReady();
```

Comment:
It will fail if the table is hidden initially. In AT dashboard, table is visible after click `Expand` button.